src/ConsoleApp/ProjectConverter.cs

```diff
@@ -481,8 +486,10 @@ private void ProcessElement(ProjectElement element, List<PackageReference> packa
                     }
                     else
                     {
-                        path = $"$(NuGetPackageRoot){path.Substring(_globalPackagesFolder.Length)}";
+                        var generatedProperty = $"$(Pkg{package.PackageId.Replace(".", "_").Replace("-", "_")})";
+                        path = $"{generatedProperty}{path.Substring(_globalPackagesFolder.Length + package.PackageId.Length + package.PackageVersion.ToFullString().Length + 1 + 1)}";

```

Comment:
This seems unreliable.  Is there a better algorithm that you can come up with or am I wrong?